contrib/hadoop-store-builder/src/java/voldemort/store/readonly/fetcher/HdfsFetcher.java

```diff
@@ -210,6 +210,36 @@ public File fetch(String sourceFileUrl,
         }
     }
 
+    /**
+     * Replace swebhdfs protocol and port in sourceFileUrl when SSL is not enabled in this node.
+     */
+    protected String modifyURLbyConfig(String sourceFileUrl) {
```

Comment:
Will do. Can I move this feature to https://github.com/voldemort/voldemort/blob/master/contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/utils/VoldemortUtils.java